content/blogposts/2020/3.18/custom-search-pages.md

```diff
@@ -15,49 +15,44 @@ This is a great resource that we hope will make these communities accessible to
 
 Here are the communities currently supported:
 
-<a style={{display: 'block'}} target="_blank" href="https://sourcegraph.com/kubernetes">
-<div style={{fontSize: '2em', textAlign: 'center', marginBottom: '-1em'}}>
-  Kubernetes
-</div>
-<div>
-<img alt="K8s search page" src="https://sourcegraphstatic.com/blog/3.18/k8s-search-page.png" />
-</div>
+<a target="_blank" href="https://sourcegraph.com/kubernetes">
+  <h2 className="text-center">Kubernetes</h2>
+  <Figure 
+    alt="K8s search page" 
+    src="https://sourcegraphstatic.com/blog/3.18/k8s-search-page.png"
+  />
 </a>
```

Comment:
I'm noticing when there's a link wrapping `<Figure />` the image alt tag isn't picked up by the screen reader b/c of how nested the image is. So on this page, these blocks read "Link: Kubernetes. Link: Go". Maybe we should re-think how we treat links with our custom Figures so alt tags get picked up properly? Or ensure a descriptive title is in each anchor tag?
On this page, I think the image should definitely be described considering what it's showing.
![image](https://user-images.githubusercontent.com/59381432/176932370-22f2152f-d0bf-4c0e-ba0a-d29308b9e580.png)

Reply:
Good point, @st0nebraker. One way we can handle this is to use a wrapper component, such as `LinkWrapper` in b0f3707. Otherwise, I think we can require writers to include a `title` on the `<a>` tags in our standards for the blog. The former ensures that links are implemented consistently, but the latter may be more simple.

If we choose to just go with guidance in the template, I can revert this last commit and add a note to the template we're creating in #https://github.com/sourcegraph/about/issues/5513 and the Handbook page. Let me know what you think!

Reply:
Updating the `Figure` sounds like the way to go, because we will have designed a11y into the components themselves. I am happy to work on it before my last day. I set up a ticket here: https://github.com/sourcegraph/about/issues/5554.

Reply:
You're awesome! Thanks @zlonko!